data-structures/stack.md

```diff
@@ -4,8 +4,8 @@
 About 1 hour
 
 ### Prerequisites
-JS
-Functional programming
+- JS
```

Comment:
@yati1998 , write JS as JavaScript and link it to:
https://github.com/Techtonica/curriculum/tree/master/javascript

Reply:
good work @yati1998